docs/src/main/sphinx/connector/jdbc-common-configurations.fragment

```diff
@@ -24,11 +24,11 @@ connector:
     - ``null``
   * - ``case-insensitive-name-matching.refresh-period``
     - Frequency with which Trino checks the name matching configuration file
-      for changes.
+      for changes. This value should be a :ref:`prop-type-duration`.
     - ``0`` (refresh disabled)
```

Comment:
The default value is still invalid. 
```suggestion
    - ``0s`` (refresh disabled)
```

Reply:
I think 0s/0m is better because it's technically the case? I feel like just putting (refresh disabled) is misleading, too.

Reply:
No, the minimum value of this property is `1ms`. It doesn't accept 0s/0m.

Reply:
Is it just not enabled until you specify a value?